CASMNET- 2278 Cilium troubleshooting doc (#6213)
Cray-HPE/docs-csm: troubleshooting/cilium_runbook.md

    ```

    Expected output resembles the following:

    ```console
    [1] 903511
    ncn-m001:~ # ℹ️  Hubble Relay is available at 127.0.0.1:4245
    ```

1. (`ncn-mw#`) Use the `hubble` command to inspect network traffic and events.

    > This example looks at traffic that is being dropped because of network policies.
    > This command will show the last 5 dropped packets along with their details.

    ```bash
    hubble observe --verdict DROPPED --last 5
    ```

    Example output:

    ```text
    Aug  7 21:40:47.320: fe80::a0ec:7cff:fe45:72b (ID:32748) <> ff02::2 (unknown) Unsupported L3 protocol DROPPED (ICMPv6 RouterSolicitation)
    Aug  7 21:40:49.515: sysmgmt-health/vmagent-vms-0-6cbf9d467c-49nxn:39932 (ID:64899) <> dvs/cray-dvs-mqtt-ss-1:15020 (ID:50714) Policy denied DROPPED (TCP Flags: SYN)
    Aug  7 21:40:50.084: sysmgmt-health/vmagent-vms-1-5d5dddd445-q9znc:60136 (ID:3549) <> dvs/cray-dvs-mqtt-ss-0:15020 (ID:50714) Policy denied DROPPED (TCP Flags: SYN)
    Aug  7 21:40:50.091: sysmgmt-health/vmagent-vms-1-5d5dddd445-jp7gf:58510 (ID:3549) <> dvs/cray-dvs-mqtt-ss-0:15020 (ID:50714) policy-verdict:none INGRESS DENIED (TCP Flags: SYN)
    ```

>     

Reviewer: ```suggestion

```

> For more information on using the Hubble CLI, see the

Reviewer: ```suggestion

For more information on using the Hubble CLI, see the
```

> For Hubble documentation, refer : [Hubble documentation](https://docs.cilium.io/en/stable/observability/hubble/hubble-cli/)

Reviewer: ```suggestion
For more information on using the Hubble CLI, see the
[Hubble CLI documentation](https://docs.cilium.io/en/stable/observability/hubble/hubble-cli/).
```
